examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -64,9 +65,27 @@
              n_jobs=1, proj=True, ax=ax, color=(0, 1, 0), picks=picks)
 
 # And now do the same with SSP + notch filtering
-raw.notch_filter(np.arange(60, 241, 60), picks=picks, n_jobs=1)
+# Pick all channels for notch since the SSP projection mixes channels together
+picks_notch = mne.pick_types(raw.info, meg=True, eeg=True)
```

Comment:
ok cool

On Sat, Jan 2, 2016 at 12:04 PM, Eric Larson <notifications@github.com>
wrote:

> In examples/time_frequency/plot_compute_raw_data_spectrum.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r48686255>:
>
> > @@ -64,9 +65,27 @@
> >               n_jobs=1, proj=True, ax=ax, color=(0, 1, 0), picks=picks)
> >
> >  # And now do the same with SSP + notch filtering
> > -raw.notch_filter(np.arange(60, 241, 60), picks=picks, n_jobs=1)
> > +# Pick all channels for notch since the SSP projection mixes channels together
> > +picks_notch = mne.pick_types(raw.info, meg=True, eeg=True)
>
> I'd get rid of it, especially if we want to push people toward filtering
> their data uniformly if possible
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r48686255>.
>